SKPhotoBrowser/SKPhotoBrowser.swift

```diff
@@ -577,7 +577,9 @@ private extension SKPhotoBrowser {
     
     func configureToolbar() {
         toolbar = SKToolbar(frame: frameForToolbarAtOrientation(), browser: self)
-        view.addSubview(toolbar)
```

Comment:
Please check for `SKPhotoBrowserOptions.displayAction` so that it would work again if we use the action sheet again.

Reply:
fb00032